client/my-sites/site-logs/components/toolbar/index.tsx

```diff
@@ -0,0 +1,33 @@
+import DateRange from 'calypso/components/date-range';
+import { useLocalizedMoment } from 'calypso/components/localized-moment';
+import './style.scss';
+
+type toolbarProps = {
+	startDateRange: number;
```

Comment:
I don't think the props needs `Range` in the name. Just `startDate` and `endDate`. Or even `start` or `startDateTime`, since the unix timestamp includes time-of-day too.